.github/workflows/build.yml

```diff
@@ -178,6 +178,18 @@ jobs:
             config: Release,
             args: -DBUILD_SHARED_LIBS=ON
           }
+        - {
+            name: "MacOSX/Dll/X64/Debug/Framework",
```

Comment:
I was unsure of how you would like this named. The frameworks are still shared libraries, so I thought I would leave it under the `Dll` convention, happy to change it though.

Reply:
Replace Dll with Framework i think is fine

Reply:
OK, sounds good.